cpp/src/slice2cpp/Gen.cpp

```diff
@@ -772,44 +773,19 @@ Slice::Gen::generate(const UnitPtr& p)
     }
 
     // Emit #include statements for any 'cpp:include' metadata directives in the top-level Slice file.
```

Comment:
Same idea, don't double check the metadata validator.
This also has the added benefit of greatly simplifying the code.